build.gradle

```diff
@@ -39,10 +41,26 @@ subprojects {
     }
 
     dependencies {
+
+        // Exclude transient dependencies from Spine
+        final def excludes = {
+            exclude group: 'com.google.protobuf', module: 'protobuf-lite'
+            exclude group: 'io.grpc'
+        }
+
         // Spine
-        compile group: 'org.spine3', name: 'spine-server-core', version: project.SPINE_VERSION, changing: true
-        compile group: 'org.spine3', name: 'spine-client-core', version: project.SPINE_VERSION, changing: true
-        compile group: 'org.spine3', name: 'spine-testutil-core', version: project.SPINE_VERSION, changing: true
+        compile group: 'io.spine', name: 'spine-base', version: project.BASE_VERSION, changing: true, excludes
```

Comment:
Please have a look at [GAE configuration](https://github.com/SpineEventEngine/gae-java/blob/master/build.gradle). We don't have as much dependencies as here.